group_vars/all.yml

```diff
@@ -0,0 +1,21 @@
+---
+# Environment specific variables
+ORG: pulibrary
+ENVIRONMENT: dev
+SERVER_ROLE: web
+
+# VPC specific variables
+VPC_NAME: '{{ ORG }}-{{ ENVIRONMENT }}'
+VPC_REGION: us-west-1
```

Comment:
Are we using us-west-1 for geographic separation from campus?  Would us-east-1 be any faster?

Reply:
this seems to be the default of our account. I am assuming it was build for DR. Will change it though.